projects/br_scs/src/Mirza/BusinessRegistry/Types.hs

```diff
@@ -67,7 +73,11 @@ instance HasKatipLogEnv BRContext where katipLogEnv = brKatipLogEnv
 instance HasKatipContext BRContext where
   katipContexts = brKatipLogContexts
   katipNamespace = brKatipNamespace
+instance HasAuthPublicKey BRContext where authPublicKey = brAuthPublicKey
```

Comment:
Drop instance functions on to a new line, e.g.
```
instance Foo a where
  foo = ...
```

Reply:
Done bca915abcc70eb9bd513e2ca9c0ace6b3f9674c2